src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  const weekDiscount = 50;
+  const threeDayDiscount = 20;
+  const dayCost = perDay * days;
+
+  if (days < 3) {
```

Comment:
what is `3`?

Reply:
ok, I understood what you mean 👌
Please, check the corrections